spec/css/selector.hrx

```diff
@@ -0,0 +1,199 @@
+// CSS spec, but they're no longer supported and should now produce errors.
+.foo /bar/ .baz {
+  a: b;
+}
+
+<===> reference_combinator/error
+Error: expected selector.
+  ,
+3 | .foo /bar/ .baz{
+  |      ^
+  '
+  input.scss 3:6  root stylesheet
+
+<===>
+================================================================================
+<===> slotted/input.scss
+::slotted(.a) {x: y}
+
+::slotted(.c.d) {x: y}
+.e {@extend .c}
+
+::slotted(.f) {x: y}
+::slotted(.g) {@extend .f}
+
+<===> slotted/output.css
+::slotted(.a) {
+  x: y;
+}
+
+::slotted(.c.d, .d.e) {
+  x: y;
+}
+
+::slotted(.f, ::slotted(.g)) {
+  x: y;
+}
+
+<===>
+================================================================================
+<===> error/attribute/modifier/no_operator/input.scss
+[a b] {c: d}
+
+<===> error/attribute/modifier/no_operator/error
+Error: Expected "]".
+  ,
+1 | [a b]{c: d}
+  |    ^
+  '
+  input.scss 1:4  root stylesheet
+
+<===> error/attribute/modifier/no_operator/error-libsass
+Error: invalid operator in attribute selector for a
+        on line 1:2 of input.scss
+>> [a b] {c: d}
+
+   -^
+
+<===>
+================================================================================
+<===> error/attribute/modifier/too_long/input.scss
+// Attribute modifiers must be single characters.
+[a=b cd] {e: f}
+
+<===> error/attribute/modifier/too_long/error
+Error: expected "]".
+  ,
+2 | [a=b cd]{e: f}
+  |       ^
+  '
+  input.scss 2:7  root stylesheet
+
+<===> error/attribute/modifier/too_long/error-libsass
+Error: unterminated attribute selector for a
+        on line 2:4 of input.scss
+>> [a=b cd] {e: f}
+
+   ---^
+
+<===>
+================================================================================
+<===> error/attribute/modifier/underscore/input.scss
+// Attribute modifiers must be ASCII alphabetical characters.
+[a=b _] {c: d}
+
+<===> error/attribute/modifier/underscore/error
+Error: expected "]".
+  ,
+2 | [a=b _]{c: d}
+  |      ^
+  '
+  input.scss 2:6  root stylesheet
+
+<===> error/attribute/modifier/underscore/error-libsass
+Error: unterminated attribute selector for a
+        on line 2:4 of input.scss
+>> [a=b _] {c: d}
+
+   ---^
+
+<===>
+================================================================================
+<===> error/attribute/modifier/digit/input.scss
+// Attribute modifiers must be ASCII alphabetical characters.
+[a=b 1] {c: d}
+
+<===> error/attribute/modifier/digit/error
+Error: expected "]".
+  ,
+2 | [a=b 1]{c: d}
+  |      ^
+  '
+  input.scss 2:6  root stylesheet
+
+<===> error/attribute/modifier/digit/error-libsass
+Error: unterminated attribute selector for a
+        on line 2:4 of input.scss
+>> [a=b 1] {c: d}
+
+   ---^
+
+<===>
+================================================================================
+<===> error/attribute/modifier/unicode/input.scss
+// Attribute modifiers must be ASCII alphabetical characters.
+[a=b ï] {c: d}
+
+<===> error/attribute/modifier/unicode/error
```

Comment:
It looks like this is failing becomes of a missing error message here.

Reply:
Fixed, thanks!